### DISCUSSION TITLE
Image of next/image puts <img> tag inside <div>

### DISCUSSION THREAD

**Discussion by shehi** [vercel/next.js#26482] [Ideas]
### Describe the feature you'd like to request

When using `Image` of `next/image`, `<img>` tags generated are placed inside a new `<div>` tags. This shouldn't be happening, as it is affecting our semantic-markup considerations. 

### Describe the solution you'd like

Please remove that extra `<div>` and put its styles to the actual `<img>` tag. Both can act as block-level tags, why introduce a new, unnecessary one?!

### Describe alternatives you've considered

None. I just stopped using `next/image` until this is fixed.

**Comment by whoanuragverma** [comment]
The extra `<div>` tag forces the browser to allocate extra space while the image is loaded, this is needed for two reasons:
1. The UX will be bit bouncy as the contents of the page will move to give space for the newly fetched image if we don't allocate space for them.
2. This also reduces Cumulative Layout Shift (CLS) and imporves the lighthouse score.

If you want to learn more about CLS you can go to this great article by [web.dev](https://web.dev/optimize-cls/)

**Reply by shehi** [reply]
I know all those things. None of which are a problem. Even junior dev with HTML knowledge knows that if you don't want things to shift around, you give size to them, images not outstanding. And next/image already is doing exactly that: requires dimensions or fill. 

At the very least maybe use FIGURE tag instead of DIV? Or parametrize what tag dev prefers to use? Again, to defend your arguments which in general aren't invalid, you are forcing your markup to us, which isn't cool at all. I hope you guys consider more flexible solution to this problem than a forced one.

**Reply by whoanuragverma** [reply]
LoL if you feel someone's forcing something on you then you can always fork their code and build your own.

https://github.com/vercel/next.js/blob/917a9acc2cd8e16a7915d0b99d2ad398ede15d65/packages/next/client/image.tsx#L547

https://github.com/vercel/next.js/blob/917a9acc2cd8e16a7915d0b99d2ad398ede15d65/packages/next/client/image.tsx#L549

Since, you wanted to use figure tags I have highlighted lines where that needs to be changed. Feel free to fork.

Peace out. ✌🏻
